doc/tenant-reference.md

```diff
@@ -29,6 +29,10 @@ Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdow
 | Master Account Credentials Secret | `masterCredentialsRef` | object | See [Master Secret](#Master-Secret) for more details | Yes |
 | Admin Secret | `passwordCredentialsRef` | object | See [Admin Secret](#Admin-Secret) for more details | Yes |
 | Tenant Credentials Secret | `tenantSecretRef` | object | See [Tenant Secret](#Tenant-Secret) for more details | No |
+| Finance Support Email             | `financeSupportEmail` | string | Finance support email address              | No           |
```

Comment:
```suggestion
| Finance Support Email             | `financeSupportEmail` | string | Finance support email address              | No          |
```

Reply:
Updated , aligned with existed ..
Thank you